src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -221,6 +222,9 @@ public long StartBackupTask(long taskId, bool isFullBackup, long? operationId =
                 throw new InvalidOperationException($"Backup task id: {taskId} doesn't exist");
             }
 
+            if (operationId == AbstractOperations<Operations.Operation>.InvalidOperationId)
```

Comment:
can we somehow avoid that Abstract<>.InvalidOperationId and move that somewhere else? To Constants maybe?